### PR TITLE
Feature/setting repo

### PR DESCRIPTION
**Pull request by Hector096**
Milestone 1: Setup React Repository.
Milestone 2: Setup Redux and create a Ducks directory structure.
Milestone 3: Add UI component for showing list of Books and a Form to add books.

### CODE REVIEW

File: src/components/BookContainer.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      <hr />
+      <h2 className="ms-5 m-2 ps-2">Add Book</h2>
+      <Form className="d-lg-flex ms-5">
+        <Col className="m-3"><Form.Control type="text" placeholder="Book title" className="p-3 me-3" /></Col>
+
+        <Col className="m-3"><Form.Control type="text" placeholder="Category" className="p-3 me-3" /></Col>
+        <Col className="m-auto">
+          <Button variant="primary" type="submit">
+            Add book
+          </Button>
```
Comment (Dipeshtwis) [inline src/components/BookContainer.js]: - [ ] As per the project requirement, there should be an author name as well along with the title. Please include that and pass it to the `Books` component. Include author input in the form as well.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,66 @@
-# Bookstore
-Bookstore is a SPA where you can manage your books more effectively then a regular library. Built with React and Redux.
+# Math Magicians
+
```
Comment (Dipeshtwis) [inline README.md]: - [ ] The title here is different from what the project is about. Please fix the title of the project.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Hector096, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: The project structure is looking good.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookContainer.js (hunk 1 above)
Comment (Hector096) [reply src/components/BookContainer.js]: Hi @Dipeshtwis ,

As per the project requirement, I can't see any author input required.

**Comment by Davidosky007** [comment]
## Hi @Hector096 👋, 

## Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 


good-job-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**